frontend/packages/noobaa-storage-plugin/src/components/buckets-card/buckets-card.tsx

```diff
@@ -58,11 +60,21 @@ const ObjectDashboardBucketsCard: React.FC<DashboardItemProps> = ({
     BucketsCardQueries.UNHEALTHY_BUCKETS_CLAIMS,
     'result',
   ]);
+  const bucketsLinksResponse = prometheusResults.getIn([
+    BucketsCardQueries.BUCKETS_LINK_QUERY,
+    'result',
+  ]);
+
+  const noobaaSystemAddress = getMetric(bucketsLinksResponse, 'system_address');
+  const noobaaSystemName = getMetric(bucketsLinksResponse, 'system_name');
+  let link = '';
+  if (!_.isNil(noobaaSystemAddress) && !_.isNil(noobaaSystemName))
+    link = `${noobaaSystemAddress}/fe/systems/${noobaaSystemName}/buckets/data-buckets`;
 
   const bucketProps: BucketsType = {
     bucketsCount: getPropsData(objectBucketsCount),
     objectsCount: getPropsData(objectsCountOnBuckets),
-    unhealthyCount: getPropsData(unhealthyBucketsCount),
+    unhealthyCount: 20 || getPropsData(unhealthyBucketsCount),
```

Comment:
Is it left by mistake?
```suggestion
    unhealthyCount: getPropsData(unhealthyBucketsCount),
```